tests/foreman/ui/test_contentview.py

```diff
@@ -88,21 +90,24 @@ def setup_to_create_cv(self, repo_name=None, repo_url=None, repo_type=None,
                 product=product,
                 unprotected=repo_unprotected,
             ).create().id
-        elif rh_repo:
+        elif rh_repos:
             # Uploads the manifest and returns the result.
             with manifests.clone() as manifest:
                 upload_manifest(org_id, manifest.content)
-            # Enables the RedHat repo and fetches it's Id.
-            repo_id = enable_rhrepo_and_fetchid(
-                basearch=rh_repo['basearch'],
-                org_id=str(org_id),  # OrgId is passed as data in API hence str
-                product=rh_repo['product'],
-                repo=rh_repo['name'],
-                reposet=rh_repo['reposet'],
-                releasever=rh_repo['releasever'],
-            )
+            for rh_repo in rh_repos:
+                # Enables the RedHat repo and fetches it's Id.
+                repo_id = enable_rhrepo_and_fetchid(
+                    basearch=rh_repo['basearch'],
+                    # OrgId is passed as data in API hence str
+                    org_id=str(org_id),
+                    product=rh_repo['product'],
+                    repo=rh_repo['name'],
+                    reposet=rh_repo['reposet'],
+                    releasever=rh_repo['releasever'],
+                )
         # Sync repository
         entities.Repository(id=repo_id).sync()
```

Comment:
I think this line should be nested inside the loop, otherwise only the last repo from `rh_repos` would be really synced.